scripts/projectData.json

```diff
@@ -0,0 +1,27 @@
+[{
+    "title": "Budget Buddy",
+    "developer": "Dan Stineback",
+    "date": "2016-04-05",
+    "projectUrl": "https://github.com/dstineback/budget-buddy",
+    "body": "<p>Final group project for 201...more info to come.</p>",
+    "category": "great"
+}, {
+    "title": "Bus Mall",
+    "developer": "Dan Stineback",
+    "date": "2016-03-29",
+    "projectUrl": "https://github.com/dstineback/bus-mall/tree/friday",
+    "body": "<p>Class project for a bus app company...more info to come</p>",
+    "category": "okay"
+}, {
+    "title": "About Me",
+    "developer": "Dan Stineback",
+    "date": "2016-03-16",
+    "projectUrl": "https://github.com/dstineback/about-me-1/tree/master",
+    "body": "<p>Class project about me...more info to come</p>",
+    "category": "rookie"
+}]
+
+
+
+
+var projectData = [];
```

Comment:
This line shouldn't be here. Run your JSON through a linter and make sure you're producing valid JSON.